README.md

```diff
@@ -92,3 +96,31 @@ jobs:
         asset_name: ${{ matrix.asset_name }}
         tag: ${{ github.ref }}
 ```
+
+Example with file_glob:
+```yaml
+name: Publish
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  build:
+    name: Publish binaries
+    runs-on: ubuntu-latest
+    steps:
+    - uses: hecrj/setup-rust-action@v1-release
```

Comment:
Should change this to main repo.

Reply:
sorry not sure what you mean, the file_glob example reuses the same steps as the previous examples

the relevant step "Upload Binaries to release" already uses the main repo
```yaml
 - name: Upload binaries to release
      uses: svenstaro/upload-release-action@v1-release
      with:
        repo_token: ${{ secrets.GITHUB_TOKEN }}
        file: target/release/my*
        tag: ${{ github.ref }}
        overwrite: true
        file_glob: true
```

Reply:
Sorry, ignore that. PR view on mobile is tricky.